src/sql/parts/modelComponents/modelComponentWrapper.component.ts

```diff
@@ -146,6 +152,7 @@ export class ModelComponentWrapper extends AngularDisposable implements OnInit {
 		// set widget styles to conform to its box
 		el.style.overflow = 'hidden';
 		el.style.position = 'relative';
+		el.style.height = '100%';
```

Comment:
do we want this for all components? wouldn't it make components like text areas take up too much space?